apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/StorageActivity.java

```diff
@@ -74,26 +74,32 @@
     private AppCenterConsumer<PaginatedDocuments<TestDocument>> mUploadApp = new AppCenterConsumer<PaginatedDocuments<TestDocument>>() {
 
         @Override
-        public void accept(PaginatedDocuments<TestDocument> documents) {            
+        public void accept(PaginatedDocuments<TestDocument> documents) {
             mAppDocumentsLoading = false;
             if (!mUserDocumentsLoading) {
-               hideProgress();
+                hideProgress();
             }
             mCurrentAppDocuments = documents;
-            updateAppDocument(documents.getCurrentPage().getItems());
+
```

Comment:
<img width="272" alt="Screen Shot 2019-04-19 at 4 37 28 PM" src="https://user-images.githubusercontent.com/623672/56448102-71dc8b80-62c1-11e9-8bfd-7b1ba598531f.png">

Please always commit using those options in Android Studio, this code is not formatted correctly.

Also we don't put blank lines without a comment.